programs/bpf_loader/src/helpers.rs

```diff
@@ -403,6 +403,9 @@ mod tests {
         .unwrap();
     }
 
+    // Ignore this test: solana_logger conflicts when running tests concurrently, 
+    // this results in the bad string length being ignored and not returning an error
+    #[ignore]
```

Comment:
alternatively would a `#[serial]` fix this too?

Reply:
Hmm, interesting, that might, I'll give it a try, feel free to merger this fix now to get CI back on track and I'll submit the `#[serial]` fix if it works